Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -767,170 +799,86 @@ private void SetupScrollingBackground()
 
-            // Adjust scale and position of our clipping box
+            // Adjust scale and position of clipping box
             switch (scrollDirection)
             {
                 case ScrollDirectionType.UpAndDown:
                 default:
 
                     // Apply the viewable area and column/row multiplier
                     // Use a dummy bounds of one to get the local scale to match;
-                    clippingBounds.size = new Vector3((clippingBounds.size.x * ItemsPerTier), (clippingBounds.size.y * ViewableArea), clippingBounds.size.z);
-                    clipBox.transform.localScale = new Bounds(Vector3.zero, Vector3.one).GetScaleToMatchBounds(clippingBounds, OcclusionScalePadding);
+                    clippingBounds.size = new Vector3((CellWidth * CellsPerTier), (CellHeight * TiersPerPage), CellDepth);
+                    ClipBox.transform.localScale = new Bounds(Vector3.zero, Vector3.one).GetScaleToMatchBounds(clippingBounds);
 
                     // Adjust where the center of the clipping box is
-                    viewableCenter.x = (clipBox.transform.localScale.x * 0.5f) - (OcclusionScalePadding.x * 0.5f) + OcclusionPositionPadding.x;
-                    viewableCenter.y = (((clipBox.transform.localScale.y * 0.5f) - (OcclusionScalePadding.y * 0.5f)) * -1) + OcclusionPositionPadding.y;
-                    viewableCenter.z = OcclusionPositionPadding.z;
+                    viewableCenter.x = ClipBox.transform.localScale.x * 0.5f;
+                    viewableCenter.y = ClipBox.transform.localScale.y * -0.5f;
+                    viewableCenter.z = 0;
+
                     break;
 
                 case ScrollDirectionType.LeftAndRight:
 
                     // Same as above for L <-> R
-                    clippingBounds.size = new Vector3(clippingBounds.size.x * ViewableArea, clippingBounds.size.y * ItemsPerTier, clippingBounds.size.z);
-                    clipBox.transform.localScale = new Bounds(Vector3.zero, Vector3.one).GetScaleToMatchBounds(clippingBounds, OcclusionScalePadding);
+                    clippingBounds.size = new Vector3(CellWidth * TiersPerPage, CellHeight * CellsPerTier, CellDepth);
+                    ClipBox.transform.localScale = new Bounds(Vector3.zero, Vector3.one).GetScaleToMatchBounds(clippingBounds);
+
 
                     // Same as above for L <-> R
-                    viewableCenter.x = (clipBox.transform.localScale.x * 0.5f) - (OcclusionScalePadding.x * 0.5f) + OcclusionPositionPadding.x;
-                    viewableCenter.y = ((clipBox.transform.localScale.y * 0.5f) - (OcclusionScalePadding.y * 0.5f) + OcclusionPositionPadding.y) * -1.0f;
-                    viewableCenter.z = OcclusionPositionPadding.z;
+                    viewableCenter.x = ClipBox.transform.localScale.x * 0.5f;
+                    viewableCenter.y = ClipBox.transform.localScale.y * -0.5f;
+                    viewableCenter.z = 0;
+
                     break;
```

Comment:
this code seems identical except the clippingBounds.size set - you could just set the size for each case and then do the other calculations afterwards to avoid code duplication